### PR TITLE
Update release docs for F36 related changes & Cockpit CI

### PR DESCRIPTION
**Pull request by M4rtinK**
Update docs to state of the art as of F36 & add a note about how to enable Cockpit CI for Fedora branches.

### CODE REVIEW

File: docs/release.rst [hunk 1]
```diff
@@ -283,7 +281,8 @@ And change content according to comments in the file.
 
 
 Then correct pykickstart version for the new Fedora release by changing all occurrences of
-the DEVEL constant imported from pykickstart for the F<version> constant, for example:
+the DEVEL constant imported from pykickstart for the F<version> constant.
+The has to be done on f<version>-release branch only. For example:
```
Comment (VladimirSlavik) [inline docs/release.rst:305]: s/The/This/

**Review by VladimirSlavik — APPROVED** [review]
LGTM except for a typo. Thanks!

File: docs/release.rst (hunk 1 above)
Comment (M4rtinK) [reply docs/release.rst]: Fixed! :)

**Comment by jkonecny12** [comment]
This is in collision with https://github.com/rhinstaller/anaconda/pull/3827 sorry for missing this PR first.

**Pull request merged**